packages/common/src/index.ts

```diff
@@ -1248,3 +1248,67 @@ export const getAxisName = ({
         ? axisName || fallbackSeriesName
         : undefined;
 };
+
+export function formatValue<T>(format: string, value: T): string | T {
```

Comment:
@rephus how come these moved out of the submodule? Was it to be able to export them?

Just in case that was the reason, you can export from the submodule (see the pattern at the top of this file)

Reply:
They were previously in the backend, but I need to use them now in the frontend. So I moved them to common

Reply:
Ah I'm so sorry! I thought it had moved in the backend. Could you move them to the `savedCharts` submodule in `common` or maybe it's own module? We're trying to reduce the size of the `index.ts` file!

Reply:
Damn, I'm trying to move it to a different package, but formatter depends on `import { Explore, fieldId, getFields, ResultRow } from '..';` so it creates a circular dependency

Reply:
Is there any way I can do import stuff directly from the package `common/src/types/formatter` rather than  `common` ? I tried but it says `You may need an additional loader to handle the result of these loaders.`

Reply:
Ah ok, if there's a circular dependency let's leave it!